pediatric_rootlets/generate_figure_rootlets_and_vertebral_spinal_levels.py

```diff
@@ -0,0 +1,223 @@
+#!/usr/bin/env python
+#
+# This script is used to generate a figure showing the variability between spinal levels and vertebrae levels.
+# The script also computes a distance from the PMJ to the middle of each spinal level and COV. Results are saved to
+# a CSV file.
+#
+# Usage:
+#     python generate_figure_rootlets_vertebrae_spinal_levels.py -i /path/to/data_processed
+#
+# Authors: Katerina Krejci, Jan Valosek
+#
+
+import os
+import sys
+import glob
+import argparse
+
+import pandas as pd
+import matplotlib as mpl
+import matplotlib.pyplot as plt
+import matplotlib.patches as patches
+import matplotlib.patheffects as pe
+
+SUBJECT_TO_AXIS = {
+    'sub-101': 1,
+    'sub-102': 2,
+    'sub-103': 3,
+    'sub-104': 4,
+    'sub-105': 5,
+}
+SUBJECT_TO_XTICKS = {
+    'sub-101': 'sub-101',
+    'sub-102': 'sub-102',
+    'sub-103': 'sub-103',
+    'sub-104': 'sub-104',
+    'sub-105': 'sub-105',
+}
+LIST_OF_LEVEL_TYPES = ['rootlets', 'vertebrae']
+XOFFSET = {'rootlets': -0.10, 'vertebrae': 0.10}
+LEVEL_TYPE_COLOR = {'rootlets': 'red', 'vertebrae': 'green'}
+SPINAL_LEVEL_TYPES_TO_LEGEND = {'rootlets': 'spinal', 'vertebrae': 'vertebral'}
+FONT_SIZE = 14
+
+
+def get_parser():
+    """
+    parser function
+    """
+
+    parser = argparse.ArgumentParser(
+        description='Generate a figure showing the variability of spinal and vertebral levels for individual subjects and spinal levels.',
+        prog=os.path.basename(__file__).strip('.py')
+    )
+    parser.add_argument(
```

Comment:
It would be great to sort subjects by age --> you can add a new input arg (e.g., `-participants-tsv`) to fetch `participants.tsv` containing a column with age

Reply:
Resolved in [1fca88d](https://github.com/ivadomed/model-spinal-rootlets/pull/53/commits/1fca88d0315602f3a0e81d344fe1ee53b7773a4b).